### ISSUE TITLE
Add "Latin Spanish" to language options 

### ISSUE THREAD

**Issue by Gonckzilla** [xenia-project/xenia#2094]
Hello, first of all thank you for this emulator, it's amazing and I love seeing how far it has come :)

The concept is very simple, currently there's only a single Spanish option (number 5 ES). But many games offer Latin Spanish voices, such as Gears of War, a game I used to play a lot in this language, so those voices have become very iconic and important to me and my relatives. 

I've tried changing my region setting to latin america (namely, CL), but the language remains the same. A Latin Spanish option is missing and its inclusion would be very appreciated!

**Comment by gibbed** [comment]
The tables of available languages and countries is what the Xbox 360 supported and is complete.

This is likely an issue of selecting both the language setting and the country setting if you're not getting the correct setup in-game.

**Comment by Gonckzilla** [comment]
I understand what you mean. I tried many combinations trying to get that latin spanish option, including the MX region which would undoubtedly use it (the lat spanish dub was produced there), but there was no luck with any. 

I thought maybe my game simply doesn't support it, but I examined the files and there are indeed two denominations for the "es" (spanish) option: ESM and ESN (ex. Human_Dialogue_esm.xxx, Human_Dialogue_esn.xxx).

I understand it's not a pressing matter, I'll probably try switching file names around and see if that tricks the game into using the latin option.

**Comment by minoleal** [comment]
Have anybody solve this issue?

**Comment by gibbed** [comment]
Just to confirm, the following settings as these values does not work?

```toml
[XConfig]
user_language = 5 # 5 is 'ES' (Spanish)
user_country = 71 # 71 is 'MX' (Mexico)
```

**Comment by Gonckzilla** [comment]
Yeah I tested this one specifically (ES-MX, by setting their respective values in the config file) quite a few times and it didn't trigger the lat dub

**Comment by gibbed** [comment]
This will be game specific, so if a particular game isn't recognizing it, it doesn't mean another won't.

**Comment by nahucirujano** [comment]
This might be really late. But did you check if your copy of the game is actually multilanguage?
Mine is, and I edited the config file as language 5 (Spanish) and 4 (AR, I assume is Argentina) and the game is in Latin Spanish.

**Comment by Sullykato** [comment]
you resolve it? lo resolviste?

**Comment by EveTrav** [comment]
Uploading SONIC THE HEDGEHOG 4 Episode II 2024-09-02 10-50-26.mp4…
